AMW_angular/io/src/app/settings/deployment-parameter/deployment-parameter.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="container">
+  <h1>Deployment Parameter Keys</h1>
+  <div class="row mt-2 py-4">
+    <div class="col-6 ms-auto">
+      <div class="input-group">
+        <input
+          id="keyName"
+          type="text"
+          class="form-control"
+          [(ngModel)]="keyName"
+          placeholder="Create new key"
+          (keyup.enter)="addKey()"
+        />
+        <button class="btn btn-primary" (click)="addKey()" [attr.data-cy]="'button-add'">Add</button>
+      </div>
+    </div>
+  </div>
+
+  <table class="table table-bordered">
+    <thead>
+    <tr>
+      <th>Name</th>
+      <th></th>
+    </tr>
+    </thead>
+    <tbody>
+    @for (paramKey of paramKeys; track paramKey) {
+    <tr>
+      <td>{{ paramKey.key }}</td>
+        <td>
+          <button class="btn" (click)="deleteKey(paramKey.key)" [attr.data-cy]="'delete-' + paramKey.key">
+            <i class="bi bi-trash"></i>
```

Comment:
The button should be styled and the IconComponent can be used for a nice styling of the icon.
similar example: https://github.com/liimaorg/liima/blob/6549fa510efe5c5245a2abe9b8466f039f7a8126/AMW_angular/io/src/app/settings/releases/releases.component.html#L52